newrelic/provider.go

```diff
@@ -127,24 +127,25 @@ func Provider() terraform.ResourceProvider {
 		},
 
 		ResourcesMap: map[string]*schema.Resource{
-			"newrelic_alert_channel":                resourceNewRelicAlertChannel(),
-			"newrelic_alert_condition":              resourceNewRelicAlertCondition(),
-			"newrelic_alert_policy":                 resourceNewRelicAlertPolicy(),
-			"newrelic_alert_policy_channel":         resourceNewRelicAlertPolicyChannel(),
-			"newrelic_application_settings":         resourceNewRelicApplicationSettings(),
-			"newrelic_dashboard":                    resourceNewRelicDashboard(),
-			"newrelic_entity_tags":                  resourceNewRelicEntityTags(),
-			"newrelic_events_to_metrics_rule":       resourceNewRelicEventsToMetricsRule(),
-			"newrelic_infra_alert_condition":        resourceNewRelicInfraAlertCondition(),
-			"newrelic_insights_event":               resourceNewRelicInsightsEvent(),
-			"newrelic_nrql_alert_condition":         resourceNewRelicNrqlAlertCondition(),
-			"newrelic_plugins_alert_condition":      resourceNewRelicPluginsAlertCondition(),
-			"newrelic_synthetics_alert_condition":   resourceNewRelicSyntheticsAlertCondition(),
-			"newrelic_synthetics_label":             resourceNewRelicSyntheticsLabel(),
-			"newrelic_synthetics_monitor":           resourceNewRelicSyntheticsMonitor(),
-			"newrelic_synthetics_monitor_script":    resourceNewRelicSyntheticsMonitorScript(),
-			"newrelic_synthetics_secure_credential": resourceNewRelicSyntheticsSecureCredential(),
-			"newrelic_workload":                     resourceNewRelicWorkload(),
+			"newrelic_alert_channel":                     resourceNewRelicAlertChannel(),
+			"newrelic_alert_condition":                   resourceNewRelicAlertCondition(),
+			"newrelic_alert_policy":                      resourceNewRelicAlertPolicy(),
+			"newrelic_alert_policy_channel":              resourceNewRelicAlertPolicyChannel(),
+			"newrelic_application_settings":              resourceNewRelicApplicationSettings(),
+			"newrelic_dashboard":                         resourceNewRelicDashboard(),
+			"newrelic_entity_tags":                       resourceNewRelicEntityTags(),
+			"newrelic_events_to_metrics_rule":            resourceNewRelicEventsToMetricsRule(),
+			"newrelic_infra_alert_condition":             resourceNewRelicInfraAlertCondition(),
+			"newrelic_insights_event":                    resourceNewRelicInsightsEvent(),
+			"newrelic_nrql_alert_condition":              resourceNewRelicNrqlAlertCondition(),
+			"newrelic_alerts_location_failure_condition": resourceNewRelicAlertsLocationFailureCondition(),
```

Comment:
This is a Synthetics specific feature, should probably have that in the resource name?

`newrelic_alerts_synthetics_multilocation_condition` ?

Reply:
I'm good with that.  I'll rename.